utils/spdx/src/main/kotlin/SpdxSimpleLicenseMapping.kt

```diff
@@ -28,37 +28,39 @@ import com.fasterxml.jackson.module.kotlin.readValue
  */
 object SpdxSimpleLicenseMapping {
     /**
-     * The raw map which associates custom license IDs with their corresponding SPDX license ID.
+     * The map of simple license names associated with their corresponding [SPDX license][SpdxLicense].
      */
-    internal val customLicenseIdsMap by lazy {
+    internal val simpleLicenseMapping by lazy {
```

Comment:
The renaming could have been a separate commit.

Reply:
I've extracted this plus a bit more to a preceding commit.